_posts/2019-03-11-nuestras_herramientas.md

```diff
@@ -0,0 +1,33 @@
+
+En algunos casos no es pertinente guardar los datos en un archivo de Excel. En proyectos complejos los datos provienen de fuentes diversas y no es prudente combinarlos en una sola tabla. En estos casos es necesario usar **bases de datos relacionales** para guardar la información sin redundancia en múltiples tablas y combinar las tablas considerando las relaciones que existen entre las diferentes variables del sistema. Los sistemas de gestión de bases de datos relacionales que más usamos son **MySQL** y **SQLite**.
+
+Una vez que los datos han sido depurados y almacenados, realizamos un análisis exploratorio de datos para familiarizarnos con su naturaleza y características. Durante esta manipulación inicial de los datos graficamos los datos crudos, elaboramos histogramas o diagramas de cajas y bigotes, y calculamos algunos estadísticos básicos. A medida de que avanzamos en esta exploración, los resultados mismos nos van sugiriendo realizar nuevas visualizaciones para observar los detalles de alguna característica que nos haya llamado la atención. Esta actividad un tanto lúdica la realizamos de forma interactiva, es decir, introducimos un comando en la computadora, la computadora nos responde con un resultado, y el resultado nos impulsa a introducir otro comando. En esta etapa, cada comando depende del resultado producido por el comando anterior. Esta interacción sucede en un intérprete de comandos como Bash o en el intérprete de Python o R (MATLAB lo usamos cada vez menos) o en un cuaderno interactivo (_notebook_) de Jupyter.
+
+Después del análisis exploratorio de datos tenemos algunas ideas sobre cuales metodologías nos podrían ayudar a responder la pregunta de conservación planteada. Los subsecuentes análisis también los hacemos en **Python** o **R**. Python y R son los lenguajes de programación más usados actualmente en Ciencia de Datos. Ambos son lenguajes gratuitos (a diferencia de MATLAB) y por ser tan populares es fácil encontrar ayuda sobre cómo usarlos. Además, Python se puede utilizar fuera del análisis de datos, por ejemplo, para desarrollar aplicaciones web. Para hacer nuestros análisis preferimos los lenguajes de programación o programas que ofrecen una **interfaz de línea de comando** (CLI) porque nos brindan la flexibilidad de ser usados tanto de forma interactiva (en la línea de comandos) como programática. Una vez satisfechos con los resultados obtenidos, transcribimos los comandos ejecutados a un guión de comandos (_script_) para reproducir el resultado de forma no interactiva. Eliminar la interactividad nos permite automatizar el proceso lo cual implica la reproducibilidad de nuestros resultados.

```

Comment:
En el párrafo anterior fue la primera vez que hablaste de Python y R, ¿no será ahí donde deberían de in en negritas?

Reply:
Listo